02-IDE_exercises/solutions/10-project/BuildersGame.py

```diff
@@ -86,12 +85,11 @@ def _set_player_height(self):
         self._o1_height = self._board[self._o1_position[0]][self._o1_position[1]]
         self._o2_height = self._board[self._o2_position[0]][self._o2_position[1]]
 
-
-    def _check_player_turn(self, from_row, from_col):
+    def _check_player_turn(self, from_row: int, from_col: int) -> bool:
         """
         returns a boolean of False if it's not the appropriate players turn
         """
-        player_referenced = None
+        player_referenced: str = None
```

Comment:
`player_referenced` is declared as a `str` but set as `None`. Either it should be declared and assigned an empty string `""` or it should be declared with the following:
```
from typing import Union

player_referenced: Union[str, None] = None
```

Reply:
you can actually use the `|` for Union now. I believe this should be `player_referenced: str | None = None`